pages/common/opam.md

```diff
@@ -0,0 +1,37 @@
+# opam
+
+> OCaml Package Manager.
+> Manage OCaml compilers, tools and libraries.
+> More information: <https://opam.ocaml.org/>.
+
+- Initialize opam interactively for first use:
```

Comment:
Is it necessary to include the word "interactively" ? The user would come to know about it while using the command itself right ? How about "Initialize opam for first use:" ?